src/pages/sung/LoginSung/InputLoginSung.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './InputLoginSung.scss';
+
+function InputLoginSung(props) {
+  return (
```

Comment:
props 구조분해할당 해서 사용하기
```suggestion
function InputLoginSung(props) {
  const { onChangeId, onChangePw } = props;
  return (
```
혹은
```suggestion
function InputLoginSung({ onChangeId, onChangePw }) {
  return (
```